test/grabURITest.coffee

```diff
@@ -0,0 +1,20 @@
+Helper = require('hubot-test-helper')
+helper = new Helper('../scripts/grabURI.coffee')
+
+expect = require('chai').expect
+co     = require('co')
+
+
+describe 'Otis', ->
+  context 'User posts URL', ->
+    room = helper.createRoom()
+    co =>
+      yield room.user.say('user1', 'http://fakewebsite.com')
```

Comment:
Forgive me if I'm just being stupid (JS is not my strong point), but what does wrapping this code with co do? I tried it without the wrapper and it seemed to work fine.